packages/lit-dev-content/src/components/litdev-github-share-button.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {signInToGithub} from '../github/github-signin.js';
+
+/**
+ * A button that shares a Playground project as a GitHub gist. If the user isn't
+ * signed into GitHub yet, they are prompted to do so.
+ *
+ * // TODO(aomarks) Implement actually sharing. This is just an early WIP.
+ * // TODO(aomarks) Show a scrim and some indication about what is happening
```

Comment:
Clarification: Not familiar with the term scrim? Is this an indication on the parent page that it's awaiting something? Or is this to prevent someone clicking on the button over and over without addressing the widow.

Also we may need to expose a way for someone to "log out". Currently the sign up is very sticky, so you wouldn't be able to change accounts.

Reply:
> Clarification: Not familiar with the term scrim? Is this an indication on the parent page that it's awaiting something? Or is this to prevent someone clicking on the button over and over without addressing the widow.

A scrim is an overlay effect to help direct the user's attention to e.g. a dialog or in this case a popup, usually by darkening or blurring the underlying content. See the darkening effect when you click on a button here: https://material-components.github.io/material-web/demos/dialog/

> Also we may need to expose a way for someone to "log out". Currently the sign up is very sticky, so you wouldn't be able to change accounts.

True, for this demo you can simulate it by going to `https://localhost:5417/reset` but in the real UI we should have a logout button.